test/router.js

```diff
@@ -400,7 +372,7 @@ describe('Router', function () {
               .end(cb)
           },
           function (cb) {
-            request(server)[method]('/zoo')
+            request(server)[method]('/bar')
```

Comment:
I was trying to figure out how this one worked before, but I am thinking this might be a weird git diff thing from a rebase. The `/bar` route should not have "hit" it would have needed the line 2 below to flip if we kept this as `zoo`. Anyway, not a really meaningful change so I am not going to try and undo what ever I missed in the merge and just push forward.